go.mod

```diff
@@ -265,7 +265,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/power-devops/perfstat v0.0.0-20210106213030-5aafc221ea8c // indirect
 	github.com/prometheus/client_model v0.2.0 // indirect
-	github.com/prometheus/common v0.32.1 // indirect
+	github.com/prometheus/common v0.33.0 // indirect
```

Comment:
Looks lows risk. Adding two opt-in features. https://github.com/prometheus/common/compare/v0.32.1...v0.33.0